### PR TITLE
ember-cli 1.13.8 -> 1.13.14

### PR DESCRIPTION
**Pull request by john-kurkowski**
Accept `ember init`'s output, except for ember-infinity-specific dependencies.

Acquires all the goodies and bugfixes in the last few releases of ember-cli.

### CODE REVIEW

**Comment by john-kurkowski** [comment]
The only output of `ember init` I wasn't sure about was config/ember-try.js. I'm not familiar with that file. Patch:

```diff
diff --git a/config/ember-try.js b/config/ember-try.js
index 85cbad5..3e88bc6 100644
--- a/config/ember-try.js
+++ b/config/ember-try.js
@@ -6,67 +6,30 @@ module.exports = {
       dependencies: { }
     },
     {
-      name: 'ember-1.10',
-      dependencies: {
-        ember: '~1.10.0',
-        'ember-data': '1.13.10'
-      }
-    },
-    {
-      name: 'ember-1.11',
-      dependencies: {
-        ember: '~1.11.3',
-        'ember-data': '1.13.10'
-      }
-    },
-    {
-      name: 'ember-1.12',
-      dependencies: {
-        ember: '~1.12.0'
-      }
-    },
-    {
-      name: 'ember-1.13',
-      dependencies: {
-        ember: '~1.13.0'
-      }
-    },
-    {
       name: 'ember-release',
       dependencies: {
-        ember: 'release',
-        'ember-data': 'release'
-      }
-    },
-    {
-      name: 'ember-release-ember-data-1.13.1x',
-      dependencies: {
-        ember: 'release',
-        'ember-data': '~1.13.11'
+        'ember': 'components/ember#release'
       },
       resolutions: {
-        ember: 'release',
-        'ember-data': '~1.13.11'
+        'ember': 'release'
       }
     },
     {
       name: 'ember-beta',
       dependencies: {
-        ember: 'beta',
-        'ember-data': 'beta'
+        'ember': 'components/ember#beta'
       },
       resolutions: {
-        ember: 'beta'
+        'ember': 'beta'
       }
     },
     {
       name: 'ember-canary',
       dependencies: {
-        ember: 'canary',
-        'ember-data': 'canary'
+        'ember': 'components/ember#canary'
       },
       resolutions: {
-        ember: 'canary'
+        'ember': 'canary'
       }
     }
   ]
```

File: .gitignore [hunk 1]
```diff
@@ -15,5 +15,3 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
-
-.DS_Store
```
Comment (hhff) [inline .gitignore]: can we keep `.DS_Store` in here plz

**Comment by hhff** [comment]
@john-kurkowski can we merge master here please!

File: .gitignore (hunk 1 above)
Comment (john-kurkowski) [inline .gitignore]: Shouldn't that be in [your global .gitignore](http://devoh.com/blog/2013/01/global-gitignore)? To keep this .gitignore focused on ember-infinity. And so you don't have to add your personal ignores to every project you work on.

Comment (kellyselden) [inline .gitignore]: @john-kurkowski I would agree, since this wouldn't affect Linux or Windows users.

**Comment by john-kurkowski** [comment]
Rebased to fix conflict.

File: .gitignore (hunk 1 above)
Comment (john-kurkowski) [inline .gitignore]: Er, was that a vote for keeping `.DS_Store` or removing it? :smile: Up to you both.

Comment (kellyselden) [inline .gitignore]: Haha sorry, I'm in favor of removing it from the file.

Comment (hhff) [inline .gitignore]: fair shake!  agree'd

**Comment by hhff** [comment]
Ahh sorry @john-kurkowski - missed this

THANKYOU

**Pull request merged**